pkg/test/echo/client/client.go

```diff
@@ -33,14 +35,23 @@ type Instance struct {
 }
 
 // New creates a new echo client.Instance that is connected to the given server address.
-func New(address string) (*Instance, error) {
+func New(address string, tlsSettings *common.TLSSettings) (*Instance, error) {
 	// Connect to the GRPC (command) endpoint of 'this' app.
 	ctx, cancel := context.WithTimeout(context.Background(), common.ConnectionTimeout)
 	defer cancel()
-	conn, err := grpc.DialContext(ctx,
-		address,
-		grpc.WithInsecure(),
-		grpc.WithBlock())
```

Comment:
why remove it?

Reply:
it's just moved down a few lines so we can conditional configure this